jdaviz/configs/imviz/plugins/footprints/footprints.py

```diff
@@ -170,6 +171,8 @@ class Footprints(PluginTemplateMixin, ViewerSelectMixin, HasFileImportSelect, Lo
 
     is_pixel_linked = Bool(True).tag(sync=True)  # plugin disabled if linked by pixel (default)
 
+    skewer_mode_active = Bool(True).tag(sync=True)

```

Comment:
this doesn't seem to be referenced anywhere, maybe it was just from a previous implementation?